src/ArchiveUtils.jl

```diff
@@ -133,18 +133,7 @@ function download_verify(url, hash, path)
         @info "Cached file found in $(path)"
         return true
     else
-        mkpath(dirname(path))
-        # `curl` doesn't automatically overwrite an existing file, delete it
-        # before attempting the download.
```

Comment:
We also want to keep the `@info` message and probably also the `rm` (I don't remember whether `Downloads.download` does it automatically)